code/modules/jobs/job_types/roguetown/adventurer/types/treasurehunter.dm

```diff
@@ -0,0 +1,57 @@
+/datum/subclass/puritan
```

Comment:
Witchhunter can probably be removed. It was a rare class during adventurer days and isn't really needed for the migrant wave.